.circleci/config.yml

```diff
@@ -32,9 +29,6 @@ jobs:
     executor: rn/linux_android
     steps:
       - checkout
-      - rn/yarn_install:
```

Comment:
I think we should also update `yarn bootstrap` so it doesn't try to install `purchaseTester`? it's in package.json